jobbergate-cli/jobbergate_cli/subapps/job_submissions/tools.py

```diff
@@ -60,11 +63,14 @@ def create_job_submission(
         cluster_name=cluster_name,
     )
 
-    if execution_directory is not None:
-        if not execution_directory.is_absolute():
-            execution_directory = Path.cwd() / execution_directory
-        execution_directory.resolve()
-        job_submission_data.execution_directory = execution_directory
+    if execution_directory is None:
+        execution_directory = Path.cwd()
+    else:
+        if execution_directory is not None:
```

Comment:
We know that this is true already because of the first `if`.

Reply:
Left over from the previous validation 🤦‍♀️